common/protob/messages-ethereum.proto

```diff
@@ -35,8 +36,9 @@ message EthereumPublicKey {
  * @next Failure
  */
 message EthereumGetAddress {
-    repeated uint32 address_n = 1;  // BIP-32 path to derive the key from master node
-    optional bool show_display = 2; // optionally show on display before sending the result
+    repeated uint32 address_n = 1;      // BIP-32 path to derive the key from master node
+    optional bool show_display = 2;     // optionally show on display before sending the result
+    optional bytes encoded_network = 3; // encoded Ethereum network, see protocol.md for details
```

Comment:
Should probably refer to `docs/common/ethereum-definitions.md` instead.

Reply:
[864ab42](https://github.com/trezor/trezor-firmware/pull/2410/commits/864ab42333e99a6b93de2707e299ef3f3fca06e6)